lib/improver/tests/ensemble_calibration/ensemble_calibration/helper_functions.py

```diff
@@ -38,9 +38,85 @@
 import iris
 from iris.coords import AuxCoord, DimCoord
 from iris.cube import Cube, CubeList
+from iris.tests import IrisTest
 import numpy as np
 
+from improver.tests.set_up_test_cubes import set_up_variable_cube
 from improver.utilities.cube_manipulation import concatenate_cubes
+from improver.utilities.warnings_handler import ManageWarnings
+
+
+IGNORED_MESSAGES = ["Collapsing a non-contiguous coordinate.",
+                    "Not importing directory .*sphinxcontrib'",
+                    "The pandas.core.datetools module is deprecated",
+                    "numpy.dtype size changed",
+                    "The statsmodels can not be imported",
```

Comment:
Hmm.. some of these feel a little concerning - perhaps we should consider documenting somewhere where/why each of these are raised?

Reply:
I've gone through these warnings now. I've removed the warnings where they are no longer relevant and added a comment to indicate where the warnings are coming from.

Reply:
Thanks 😄